src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -228,6 +228,31 @@ def update(
         ORMRun.get(name).update(display_name=display_name, description=description, tags=tags, **kwargs)
         return self.get(name)
 
+    @monitor_operation(activity_name="pf.runs.delete", activity_type=ActivityType.PUBLICAPI)
+    def delete(
+        self,
+        runs: Union[str, Run, List[str], List[Run]],
+        **kwargs,
+    ) -> None:
+        """Delete run permanently.
+
+        :param names: run names to delete
+        :param kwargs: other fields to update, fields not supported will be directly dropped.
+        :return: None
+        """
+        if not isinstance(runs, list):
+            runs = [runs]
+        valid_runs = []
+        for run in runs:
+            run = Run._validate_and_return_run_name(run)
+            valid_runs.append((run, LocalStorageOperations(run=self.get(run))))
+        for run, valid_run in valid_runs:
+            valid_run.delete()
```

Comment:
what's our behavior if we deleted the main flow run of eval run?
Will our run visulize experience met any issue?

Reply:
let me check.

Reply:
After some discussion with @zhengfeiwang .Evaluation run will have a lineage id point to this main flow run.
And behavior will not break, since all needed data of eval run is saved in eval run, and no other properties is pointing to main flow run.